src/geoip_cache.rs

```diff
@@ -27,15 +28,20 @@ pub struct GeoIPCache {
     cache_file: PathBuf,
     /// Cache entries.
     cache: Arc<RwLock<GeoCache>>,
+    /// How many days to keep the cache entries.
+    keep_in_cache_days: u16,
```

Comment:
just asking: is there a particular reason that this is a `u16`?  And not u32, or even u64 (where would not need the cast below).

Reply:
I (at least :) ) try to give data most accurate types from the logic point of view - I assumed that the reasonable values for this factor is no more than 365 (max year in cache - that's still long) so it fits in u16. I thought it could be surprising for someone if days count could be so big as u64 (then someone could ask question - why so big to keep values under 365). 
So if there are no other significant factors (like performance - but then there are often no rules :) ) I try to make data models or its parts to follow that accuracy rule.